### PR TITLE
build, pkgs: Bump containerd to 1.6.36 and pause to 3.10

### PR DESCRIPTION
**Pull request by eg-ayoub**

### CODE REVIEW

File: packages/redhat/common/containerd.spec [hunk 1]
```diff
@@ -55,423 +55,531 @@ Requires:       (runc < 1:1.1.3 or runc > 1:1.1.3)
 # From github.com/containerd/containerd repository, checkout the corresponding version and run:
```
Comment (eg-ayoub) [inline packages/redhat/common/containerd.spec:55]: add this to BUMPING.md

Comment (eg-ayoub) [reply packages/redhat/common/containerd.spec:55]: "take a look at spec file to update go list"

File: packages/redhat/common/containerd.spec [hunk 2]
```diff
@@ -545,6 +653,9 @@ install -D -p -m 0644 %{S:2} %{buildroot}%{_sysconfdir}/containerd/config.toml
 
 
 %changelog
+* Fri Oct 18 2024 Ayoub Nasr <ayoub.nasr@scality.com> - 1.6.36
```
Comment (TeddyAndrieux) [inline packages/redhat/common/containerd.spec]:
```suggestion
* Fri Oct 18 2024 Ayoub Nasr <ayoub.nasr@scality.com> - 1.6.36-1
```

File: buildchain/buildchain/versions.py [hunk 3]
```diff
@@ -27,7 +27,7 @@
 
 CALICO_VERSION: str = "3.29.0"
 SALT_VERSION: str = "3002.9"
-CONTAINERD_VERSION: str = "1.6.35"
+CONTAINERD_VERSION: str = "1.6.36"
 
 CONTAINERD_RELEASE: str = "2"
```
Comment (TeddyAndrieux) [inline buildchain/buildchain/versions.py]:
```suggestion
CONTAINERD_RELEASE: str = "1"
```

**Review by TeddyAndrieux — COMMENTED** [review]
Just 2 nitpick

**Review by TeddyAndrieux — DISMISSED** [review]

**Review by TeddyAndrieux — APPROVED** [review]

**Comment by m4nch0t** [comment]
clic

**Pull request merged**